<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detection</class>
 <widget class="QWidget" name="detection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="autuModeBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="5" column="1">
         <widget class="QLineEdit" name="confidenceEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>0.5</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>nms</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>置信度</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="modeLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>[模型选择] </string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QLineEdit" name="nmsEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>0.5</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="autoMode">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>auto Mode</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="ModeSelection">
          <item>
           <property name="text">
            <string>yolov8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>yolov12</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="inferAgain">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    border: 1px solid #c0c0c0;
    border-radius: 5px;
    background: #f0f0f0;
}
QPushButton:hover {
    background: #e0e0e0;
}
QPushButton:pressed {
    background: #d0d0d0;
}</string>
          </property>
          <property name="text">
           <string>重新推理</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="colorBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="targetColor">
          <property name="text">
           <string>[目标颜色]  </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="red">
          <property name="text">
           <string>red</string>
          </property>
          <property name="shortcut">
           <string>1</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">colorSelection</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QRadioButton" name="blue">
          <property name="text">
           <string>blue</string>
          </property>
          <property name="shortcut">
           <string>2</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">colorSelection</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="colorSave">
          <property name="text">
           <string>save color</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="tagetBox">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
  font-weight: bold;
  margin-top: 12px;
  padding-top: 15px;
}</string>
     </property>
     <property name="title">
      <string>Target</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QListWidget" name="labelList">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    border: 1px solid #c0c0c0;
    border-radius: 5px;
    background: #f0f0f0;
    padding: 5px;
}
QPushButton:hover {
    background: #e0e0e0;
}
QPushButton:pressed {
    background: #d0d0d0;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="createLabel">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    border: 1px solid #c0c0c0;
    border-radius: 5px;
    background: #f0f0f0;
    padding: 5px;
}
QPushButton:hover {
    background: #e0e0e0;
}
QPushButton:pressed {
    background: #d0d0d0;
}</string>
        </property>
        <property name="text">
         <string>创建标签</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="colorSelection"/>
 </buttongroups>
</ui>
